### ISSUE TITLE
Always check what pages?

### ISSUE THREAD

**Issue by mgifford** [w3c/wai-wcag-em#127]
If a site has these, they should be checked with every WCAG-EM report:

- Home
- Search (and results)
- Accessibility
- Contact us 
- Login

There is nothing worse than an inaccessible home page or accessibility page.
